emoji-contest-25.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: fr
+title: emoji-contest-25
+author: dismad
+date: January 12, 2025
+amount: 5000 VTC
+milestones:
+  - name: Emoji Contest Community Fund
+    funds: 100%
+    done:
+    status: unfinished
+payouts:
+  - date:
+    amount:
+  - date:
+    amount:
+---
+
+
+
+
+# Emoji-contest-25
+
+```
+* Welcome to the 5th annual Vertcoin emoji contest 
+* The event will run from May 1, 2025 thru May 31, 2025
+* Everyone can submit up to 3 Emojis, 3 Stickers, and 3 Wallpapers in total
+* For Emojis every graphic format is fine [JPG,PNG,SVG,GIF, ... etc. ], preferable in 128x128 [SIZE LIMIT OF 256KB]
+* For Stickers only use graphic formats [APNG, Lottie, or PNG], Image Resolution 320x320 [SIZE LIMIT OF 500KB]
+* For Wallpapers we are looking to add to our server banner background and server invite background. 16:9 [960x540] and 16:9[1920x1080]
+* Please use common sense and follow the server rules with your submissions
+* If you want to replace a submission, just delete the one you want to remove and replace it with a new one
+* Each approved submission will earn .33 vtc, regardless if it is not selected as a finalist  [ Max 1 vtc per category / user] ( 3 vtc max)
+* Finalists will be reviewed by staff and the top 3 will be selected by vote [VTCrookie and above]
```

Comment:
Specify in primary details that emojis submitted may be used on Vertcoin Discord server

Reply:
Also provide link to Vertcoin Discord server